scheduler/src/cook/mesos/api.clj

```diff
@@ -385,7 +415,7 @@
 ;;;
 ;;; On GET; use repeated job argument
 (defn job-resource
-  [conn fid task-constraints]
+  [conn fid task-constraints auth-config]
```

Comment:
Instead of passing auth-config, pass a partial of `is-authorized` with auth-config already passed in.